packages/kbn-i18n/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
__note:__ it seems we generate types for `.test.ts/tsx` as well :/ But if you don't have any easy solution we can live with it for now.